package.json

```diff
@@ -18,6 +18,12 @@
       "hash": "QmT8rehPR3F6bmwL6zjUN8XpiDBFFpMP2myPdC6ApsWfJf",
       "name": "go-base58",
       "version": "0.0.0"
+    },
+    {
+      "author": "whyrusleeping",
+      "hash": "QmNT11RM74otj4AmK3LEjSm1qN3gZuMho5Nh9eg4xXHFGV",
+      "name": "sha3",
```

Comment:
I don't like the fact that the package is called sha3 as it is keccak. Any way to fix it? I don't think the pkgname is bound to directory name, is it?